Sources/PayPalNativeCheckout/PayPalClient.swift

```diff
@@ -1,86 +1,78 @@
 import UIKit
-
-@_implementationOnly import PayPalCheckout
-
+import PayPalCheckout
 #if canImport(PaymentsCore)
 import PaymentsCore
 #endif
 
 /// PayPal Paysheet to handle PayPal transaction
+/// encapsulates instance to communicate with nxo
 public class PayPalClient {
```

Comment:
Also, I just saw @jaxdesmarais's comment regarding the naming of this file. I notice that our other PayPal Checkout client (for the web flow) is called `PayPalWebCheckoutClient`. I agree that this is a little confusing and their naming should be consistent.

Right now we have
`PayPalWebCheckout.PayPalWebCheckoutClient` and `PayPalNativeCheckout.PayPalClient`

Reply:
Also side note - can someone fill me in the background for splitting our PayPal flows into 2 modules on Northstar (native vs web)? Did we discuss consolidating?

Reply:
Yeah it's `PayPalNativeCheckout`/`PayPalClient` and `PayPalWebCheckout`/`PayPalWebCheckoutClient` on Android we'll have to rename in the future too.

I think the reasoning was we were able to implement web checkout faster initially, and to offer support for some merchants who want a hyper-optimized app with only the PayPal web flow.

Might not have been the exact reasons but I remember those coming up in conversation.

Reply:
@scannillo agree, naming is incosistent, is it good time to change, or let's keep it consistent with android?

Reply:
Ah gotcha - you mean hyper-optimized to not include the PPNXO binary.

Makes sense to me to tackle it alongside Android in a later PR 🏈